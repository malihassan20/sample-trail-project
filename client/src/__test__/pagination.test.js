import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import CustomPagination from '../components/pagination';

test('CustomPagination component renders without crashing', () => {
  render(<CustomPagination />);
});

it('CustomPagination component matches the snapshot', () => {
  const tree = renderer.create(<CustomPagination />).toJSON();
  expect(tree).toMatchSnapshot();
});
