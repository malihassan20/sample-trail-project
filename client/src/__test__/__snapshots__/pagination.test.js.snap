// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomPagination component matches the snapshot 1`] = `
<nav
  aria-label="Page navigation example"
>
  <ul
    className="pagination flex flex-row justify-content-end w-full"
  >
    <li
      className="page-item disabled"
    >
      <a
        aria-disabled="true"
        className="page-link"
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          «
        </span>
      </a>
    </li>
    <li
      className="page-item disabled"
    >
      <a
        aria-disabled="true"
        className="page-link"
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          ‹
        </span>
      </a>
    </li>
    <li
      className="page-item active"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          1
        </span>
      </a>
    </li>
    <li
      className="page-item disabled"
    >
      <a
        aria-disabled="true"
        className="page-link"
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          ›
        </span>
      </a>
    </li>
    <li
      className="page-item disabled"
    >
      <a
        aria-disabled="true"
        className="page-link"
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          »
        </span>
      </a>
    </li>
  </ul>
</nav>
`;
